templates/trivy-step.tmpl

```diff
@@ -7,7 +7,7 @@ steps:
           exit-code: {{ .TPConfig.ExitCode }}
           {{- end }}
           {{- if .TPConfig.Timeout}}
-          timeout : {{ .TPConfig.Timeout }}
+          timeout : "{{ .TPConfig.Timeout }}"
```

Comment:
nit: timeout should be of time.Duration type and then should not require `"` here, but we change that later

Reply:
Trivy accepts  "Timeout"  value as a string.